fidelisnetwork.json

```diff
@@ -0,0 +1,1073 @@
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.time",
+                    "example_values": [
+                        "Medium"
+                    ],
+                    "column_name": "Alert time",
+                    "column_order": 2,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.summary",
+                    "example_values": [
+                        "Medium"
+                    ],
+                    "column_name": "Summary",
+                    "column_order": 3,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.alert_id",
+                    "data_type": "string",
+                    "contains": [
+                        "alert id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.aggAlertCounts.*.aggAlertId",
+                    "example_values": [
+                        14
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aggAlertCounts.*.alertCount",
+                    "example_values": [
+                        282
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.anomalyAlertInfo",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.assets.alertId",
+                    "example_values": [
+                        4086
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.assets.asset.assetId",
+                    "example_values": [
+                        8
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.assets.asset.compGroupId",
+                    "example_values": [
+                        0
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.assets.asset.coverageScore",
+                    "example_values": [
+                        2
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.assets.asset.decryptEnd",
+                    "data_type": "string"
```

Comment:
Do we have some example values for these fields as well?

Reply:
No, We haven't get any values for decryptEnd. We only receiving `null` value for it. Hence, I didn't add example value.